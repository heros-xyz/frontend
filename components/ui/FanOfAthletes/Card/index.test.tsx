import { render } from "@testing-library/react";
import YourAthleteCard from ".";

const item = {
  avatar: "https://cdn.iconscout.com/icon/free/png-256/avatar-370-456322.png",
  name: "Louise Peel",
  description: "Bronze Tier Subscribed.",
  joinedDate: "12/12/2002",
};

it("renders homepage unchanged", () => {
  const { container } = render(<YourAthleteCard item={item} />);
  expect(container).toMatchSnapshot();
});
