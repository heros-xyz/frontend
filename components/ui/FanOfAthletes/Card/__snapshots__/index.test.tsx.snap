// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="css-1vl0c7b"
  >
    <div
      class="css-70qvj9"
    >
      <li
        class="chakra-wrap__listitem css-1yp4ln"
      >
        <img
          alt="user-avatar"
          class="chakra-image__placeholder css-46n8jl"
          src="https://via.placeholder.com/50"
        />
      </li>
      <div
        class="css-131wcsq"
      >
        <h2
          class="chakra-heading css-2t2s4a"
        />
        <span
          class="chakra-text css-1h4wf4v"
        >
          Bronze Tier
        </span>
        <div
          class="css-14ps5fr"
        >
          <svg
            class="chakra-icon css-1qfkg6w"
            focusable="false"
            viewBox="0 0 17 17"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M8 5.83333V8.5L10 10.5M14 8.5C14 11.8137 11.3137 14.5 8 14.5C4.68629 14.5 2 11.8137 2 8.5C2 5.18629 4.68629 2.5 8 2.5C11.3137 2.5 14 5.18629 14 8.5Z"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
          <p
            class="chakra-text css-125bvfe"
          >
            Joined: 
            Mar 2023
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
